upgrading/earlier/whats-new-2.0.rst

```diff
@@ -495,7 +495,7 @@ Other bug fixes
 *  Isolate non-standard attributes on MathML elements so that they don't
    interfere with the inner workings of MathJax.
 
-*  Properly handle width of border and padding in merrors in `HTML-CSS`
+*  Properly handle width of border and padding in errors in `HTML-CSS`
```

Comment:
This comment is about the `merror` elements, so should remain as it was originally.

Reply:
```suggestion
*  Properly handle width of border and padding in merrors in `HTML-CSS`
```